<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Open</class>
 <widget class="QWidget" name="Open">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Welcome to Sandbox</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/sandbox.png</normaloff>../icons/sandbox.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>301</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width : 1px;border-color : #2C2C2C;background-color : #393939;color:#BBBBBB;
font: 10pt &quot;Microsoft Sans Serif&quot;;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="toolButton_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>45</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>135</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>225</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>315</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>300</width>
        <height>45</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton:hover{border-style:none;background-color:#4B6EAF}
QToolButton{border-style:none}
</string>
      </property>
      <property name="text">
       <string>123</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>501</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#BBBBBB;
font: 9pt &quot;Microsoft Sans Serif&quot;;
background-color:#45474A;
border-style:none;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>244</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft Sans Serif</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton:hover{border-color:#45474A;color:#AB5C25}</string>
         </property>
         <property name="text">
          <string> Open Environment Folders</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/文件夹-开 (1).png</normaloff>../icons/文件夹-开 (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton:hover{border-color:#45474A;color:#AB5C25}</string>
         </property>
         <property name="text">
          <string> Open EXE</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/加号1.png</normaloff>../icons/加号1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_3">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton:hover{border-color:#45474A;color:#AB5C25}</string>
         </property>
         <property name="text">
          <string>  Manual Input</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/输入1.png</normaloff>../icons/输入1.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_4">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton:hover{border-color:#45474A;color:#AB5C25}</string>
         </property>
         <property name="text">
          <string> Run In Sandbox</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/运行1.png</normaloff>../icons/运行1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>241</width>
        <height>30</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{background-color:#FFFFFF;color:#000000}</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>256</width>
       <height>256</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>27</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="text">
      <string>正在分析中</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_Open_Environment_Folders">
   <property name="text">
    <string> Open Environment Folders</string>
   </property>
  </action>
  <action name="action_Open_EXE">
   <property name="text">
    <string> Open EXE</string>
   </property>
  </action>
  <action name="action_Manual_Input">
   <property name="text">
    <string>  Manual Input</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string> Run</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>action_Open_Environment_Folders</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>action_Open_EXE</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>action_Manual_Input</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_4</sender>
   <signal>clicked(bool)</signal>
   <receiver>action_Run</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
